public/fastboot-fetch.js

```diff
@@ -1,10 +1,20 @@
+/* globals define FastBoot */
 (function() {
   define('fetch', ['exports'], function(self) {
-    var fetch = FastBoot.require('node-fetch');
-    self['default'] = fetch;
-    self['Headers'] = fetch.Headers;
-    self['Request'] = fetch.Request;
-    self['Response'] = fetch.Response;
+    var AbortControllerPolyfill = FastBoot.require('abortcontroller-polyfill/dist/cjs-ponyfill');
```

Comment:
Do we need our Fastboot users to also additionally import/require the `abortcontroller` polyfil, or does this happen automagically? If so, can you briefly explain how?

Reply:
Just like with browser implementation :

```js
import fetch, { AbortController } from 'fetch';
```

I will add a paragraph to README to clarify